src/ray/raylet/lib/python/raylet_extension.cc

```diff
@@ -93,32 +88,31 @@ static PyObject *PyLocalSchedulerClient_fetch_or_reconstruct(PyObject *self,
     }
     object_ids.push_back(object_id);
   }
-  int ret = local_scheduler_fetch_or_reconstruct(
-      reinterpret_cast<PyLocalSchedulerClient *>(self)->local_scheduler_connection,
-      object_ids, fetch_only, current_task_id);
-  if (ret == 0) {
+  auto status =
+      self->raylet_client->FetchOrReconstruct(object_ids, fetch_only, current_task_id);
+  if (status.ok()) {
     Py_RETURN_NONE;
   } else {
     std::ostringstream stream;
-    stream << "local_scheduler_fetch_or_reconstruct failed: "
-           << "local scheduler connection may be closed, "
-           << "check raylet status. return value: " << ret;
+    stream << "[RayletClient] FetchOrReconstruct failed: "
+           << "raylet client may be closed, check raylet status. error message: "
+           << status.ToString();
```

Comment:
FYI. I have a PR. If this message changes, the test should be updated accordingly.

Reply:
https://github.com/ray-project/ray/pull/3518